01 - Community/People/dy-sh.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- Dmitry Savosh
+tags:
+- 
+publish: true
+---
+
+# Dmitry Savosh
+
+- GitHub: [dy-sh](https://github.com/dy-sh/) ^github
+<!-- - Discord: `@` ^discord-->
+- Website: <https://github.com/derwish-pro/> ^website
+<!-- - [[Publish sites|Publish site]]: ^publish-->
+
+%% Feel free to add a bio below this comment %%
+
+
+## Author of
+
+%% Begin Hub: Released contributions %%
+### Plugins
+- [[code-block-from-selection|Code block from selection]]
+- [[remember-cursor-position|Remember cursor position]]
+- [[consistent-attachments-and-links|Consistent attachments and links]]
+- [[unique-attachments|Unique attachments]]
+- [[find-and-replace-in-selection|Find and replace in selection]]
+
```

Comment:
@argenos  Note with this: the `01 - Community/People/dy-sh.md` file appears to indicate a rename of an exiting author, given the number of existing plugins listed...

I'll deal with this as a separate PR...

Reply:
See Issue #261 and PR #262 for implementing this.